qa/tasks/cephfs/test_volumes.py

```diff
@@ -333,6 +333,33 @@ def test_volume_rm_arbitrary_pool_removal(self):
         for pool in vol_status["pools"]:
             self.assertNotIn(pool["name"], pools)
 
+    def test_volume_rm_when_mon_delete_pool_false(self):
+        """
+        That the volume can only be removed when mon_allowd_pool_delete is set
+        to true and verify that the pools are removed after volume deletion.
+        """
+        try:
+            self._raw_cmd("config", "set", "mon", "mon_allow_pool_delete", "false")
```

Comment:
Sorry one more suggestion. Let's use the new config set interface:

`self.config_set("mon", "mon_allow_pool_delete", "false")`

and move this line above the `try:`

Reply:
I should add: the reason we use this new interface is because we want the config changes to be undone by the test infrastructure between tests. i.e. we don't want `mon_allow_pool_delete=True` to persist across tests.